piet-coregraphics/src/text.rs

```diff
@@ -49,7 +50,15 @@ struct SharedTextState {
 
 struct TextState {
     collection: FontCollection,
-    family_cache: HashMap<String, Option<FontFamily>>,
+    family_cache: AssociativeCache<
+        String,
+        Option<FontFamily>,
+        Capacity1024,
+        HashFourWay,
+        RoundRobinReplacement,
+    >,
```

Comment:
I might suggest a type alias, like:

```rust
type Cache<K, V> = AssociativeCache<K, V, Capacity1024, HashFourWay, RoundRobinReplacement>;
```

Reply:
Another question here is whether `1024` is the right capacity; in general I would expect far fewer font objects to be in use than that, but I don't have strong opinions. I guess the only downside of a larger cache is potentially using more memory than necessary holding on to a bunch of stale font objects, although this does not seem particularly likely for our current use-cases.

Personally though I think I would pick a smaller capacity, like maybe as small as `64`? It just isn't very likely that there are more than 64 different fonts in use at the same time. 🤷

Reply:
Those are all very good points, I wasn't completely sure about the size.